src/definitions.ts

```diff
@@ -1,9 +1,18 @@
+import { User, Authentication } from "./user";
+
 declare module "@capacitor/core" {
   interface PluginRegistry {
     GoogleAuth: GoogleAuthPlugin;
   }
 }
 
+/** On iOS, refresh returns an extra field */
+interface RefreshData extends Authentication {
+  refreshToken?: string;
+}
+
 export interface GoogleAuthPlugin {
-  signIn(options: { value: string }): Promise<{value: string}>;
+  signIn(): Promise<User>;
```

Comment:
User types are not consistent with the Type returned by the Native Java component. 
![image](https://user-images.githubusercontent.com/5033246/91997685-5fbd8a80-ed08-11ea-8005-7e904f22432d.png)
![image](https://user-images.githubusercontent.com/5033246/91997708-63e9a800-ed08-11ea-8e09-ba335e037ccd.png)

authentication object to be specific

Reply:
So the proper fix is to change the Java to match, right?

Reply:
Probably, I messaged in slack the other day, a PR exists already to bring Java in line that looks good… it's been waiting to be merged for months though.

Reply:
Feeling more and more like full-blown fork territory.